app/views/events/show.html.erb

```diff
@@ -112,18 +112,20 @@
   <% end %>
 
   <div class="col-xs-12 col-sm-6">
-    <% if @recent_hangout.try :live? %>
+    <%  if @recent_hangout.try :live?  %>
+      <% show_live_events %>
```

Comment:
rather than setting an instance variable here, might it be simpler to have the show_live_events return a boolean?

Reply:
ah, I see you are setting the message as well ... hmmm

Reply:
if we remove that we can make the helper method much simpler ...